contrib/scripts/unloaddb.sh

```diff
@@ -0,0 +1,387 @@
+#!/bin/bash
+#
+#
+#  Copyright 2016 CUBRID Corporation
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+verbose="no"   # set 'yes' for verbose mode
+max_num_proc=16
+num_proc=8
+table_size=()
+table_selected=()
+num_tables=0
+slot_selected=0
+# following two variable are depend on max_num_proc
+slot_size=(0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)
+num_tables_slot=(0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)
+database=""
+user="-u dba"
+pass=""
+total_pages=0
+filename=""
+target_dir=$(pwd)
+from_file=0
+num_args_remain=0
+
+slot=()
+
+function show_usage ()
+{
+         echo "Usage: $0 [OPTIONS] [database]"
+         echo " OPTIONS"
+         echo "  -t arg  Set number of parallel process; default 8"
+         echo "  -i arg  input FILE of table names; default: dump all classes"
+         echo "  -u arg  Set database user name; default dba"
+         echo "  -p arg  Set dbuser password"
+         echo "  -D arg  Set directory for unloaddb output dir/files"
+         echo "  -v      Set verbose mode on"
+
+         echo ""
+         echo " EXAMPLES"
+         echo "  $0 -t 4 -v demodb          # unload all tables in demodb"
+         echo "  $0 -i file demodb          # unload tables listed in file in demodb"
```

Comment:
The '-i' option are based on 'target option' folder.
it would be better based on 'shell file for execute'. 
an error occurs when input as in the example below.  (target file is in the same path as execute file).
(e.g. ./unloaddb.sh -u dba -t 17 -i ./search -v -D aaa demodb -> ERROR)
I think users will think so.

if not, explain to enter the absolute path in 'show usage'.

Reply:
Don't you need an installation for engine release?
If necessary, add it to CMAKE file.

Reply:
You right, target should be applied to unloaddb output only, -i option should be applied the current.
Thank for your comment!!
Fixed it.